cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java

```diff
@@ -2904,6 +2934,82 @@ private String getGroupingRulesConfFile() throws IOException {
         reader.close();
         return groupingRulesConfFile;
     } // getGroupingRulesConfFile
+    
+    private String getNameMappingsConfFile() throws IOException {
+        if (!configurationFile.exists()) {
+            return "404 - Configuration file for Cygnus not found. Details: "
+                    + configurationFile.toString();
+        } // if
+
+        BufferedReader reader = new BufferedReader(new FileReader(configurationFile));
+        String line;
+
+        while ((line = reader.readLine()) != null) {
+            if (!line.startsWith("#")) {
+                if (line.contains("name_mappings_conf_file")) {
+                    String[] splits = line.split("=");
+                    nameMappingsConfFile = splits[1].replaceAll(" ", "");
+                    break;
+                } // if
+            } // if
+        } // while
+
+        reader.close();
+        return nameMappingsConfFile;
+    } // getGroupingRulesConfFile
+    
+    private void loadNameMappings() {
+        String nameMappingsStr = null;
+        
+        // Read the Json string from the configuration file
+        try {
+            nameMappingsStr = JsonUtils.readJsonFile(nameMappingsConfFile);
+            LOGGER.debug("[nmi] Reading name mappings, Json read: " + nameMappingsStr);
```

Comment:
These "[nmi]" strings were added in `NGSINameMappingsInterceptor` in roder to say "these are logs written by `NGSINameMappingsInterceptor`". In this class, they are not necessary.